includes/shipping/free-shipping/class-wc-shipping-free-shipping.php

```diff
@@ -218,4 +191,39 @@ public function calculate_shipping( $package = array() ) {
 			)
 		);
 	}
+
+	/**
+	 * Enqueue JS to handle free shipping options.
+	 *
+	 * Static so that's enqueued only once.
+	 */
+	public static function enqueue_admin_js() {
+		if ( is_admin() ) {
```

Comment:
Since is already firing inside `admin_footer` we don't need to check for `is_admin()`.

Reply:
Belts and braces, you know... :)